packages/mds-provider-service/repository/entities/mds-provider-entity.ts

```diff
@@ -0,0 +1,26 @@
+import { IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { Nullable } from '@mds-core/mds-types'
+import { Column, Entity } from 'typeorm'
+
+@Entity('mds_providers')
+export class MDSProviderEntity extends IdentityColumn(RecordedColumn(class {})) {
+  @Column('uuid', { primary: true })
+  provider_id: string
+
+  @Column('varchar', { length: 255 })
+  provider_name: string
+
+  @Column('varchar', { length: 255, nullable: true })
+  url: Nullable<string>
+
+  @Column('varchar', { length: 255, nullable: true })
+  mds_api_url: Nullable<string>
+
+  @Column('varchar', { length: 255, nullable: true })
+  gbfs_api_url: Nullable<string>
+
```

Comment:
From the current config map, @EthanSherr requested:
```suggestion
  @Column('varchar', { array: true, length: 31 })
  provider_types: String[]
```